src/components/loaders/LayerLoader.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import EarthEngineLoader from './EarthEngineLoader.js'
+import EventLoader from './EventLoader.js'
+import ExternalLoader from './ExternalLoader.js'
+import FacilityLoader from './FacilityLoader.js'
+import OrgUnitLoader from './OrgUnitLoader.js'
+import ThematicLoader from './ThematicLoader.js'
+import TrackedEntityLoader from './TrackedEntityLoader.js'
+
+const layerType = {
+    earthEngine: EarthEngineLoader,
+    event: EventLoader,
+    external: ExternalLoader,
+    facility: FacilityLoader,
+    orgUnit: OrgUnitLoader,
+    thematic: ThematicLoader,
+    trackedEntity: TrackedEntityLoader,
+}
+
+const LayerLoader = ({ config, onLoad }) => {
+    const Loader = layerType[config.layer]
+
+    if (!Loader) {
+        return null
```

Comment:
should this be a warning or alert or something?

Reply:
Added the same console log as we had: https://github.com/dhis2/maps-app/blob/master/src/loaders/layers.js#L29